doc/source/tutorial/stats/continuous_arcsine.rst

```diff
@@ -34,4 +34,11 @@ Defined over :math:`x\in\left[0,1\right]`.  To get the JKB definition put :math:
 
      l_{\mathbf{x}}\left(\cdot\right)=N\log\pi+\frac{N}{2}\overline{\log\mathbf{x}}+\frac{N}{2}\overline{\log\left(1-\mathbf{x}\right)}
 
+References
+----------
+
+.. [1] Norman Johnson, Samuel Kotz, and N. Balakrishnan, Continuous Univariate Distributions, second edition, Volumes I and II, Wiley & Sons, 1994.
```

Comment:
```suggestion
- Norman Johnson, Samuel Kotz, and N. Balakrishnan, Continuous Univariate Distributions, second edition, Volumes I and II, Wiley & Sons, 1994.
```
Other distributions use just "-" but I'm curious whether this will render properly.

Reply:
This causes the CI errors because I didn't refer to the references in the text.